src/camera.py

```diff
@@ -0,0 +1,264 @@
+"""
+camera
+
+Author: Steven Dropiewski
+"""
+import numpy as np
+import time
+
+from picar import filedb
+from picar.SunFounder_PCA9685 import Servo
+import picar
+
+ABSOLUTE = 1
+"""The value set to identify absolute degrees."""
+RELATIVE = 0
+"""The value set to identify relative degrees."""
+
+
+class Camera:
+    """
+    Creates a camera object that can be panned and tilted.
+
+    The camera can pan in [0, 180] degrees and can tilt in [0, 150] degrees.
+    The camera operates in absolute degrees by default, but can be specified to
+    work in realtive degrees during object creation.
+
+    :param bus_number: The bus number of the servos.
+    :type bus_number: int
+
+    :param pan_channel: Pan servo's I2C channel.
+    :type pan_channel: int
+
+    :param tilt_channel: Tilt servo's I2C channel.
+    :type tilt_channel: int
+
+    :param camera_type: Camera.ABSOLUTE or Camera.RELATIVE angles.
+    :type camera_type: int
+    """
+
+    MAX_PAN_DEGREE = 180
+    """The maximum value that the camera can pan."""
+    MAX_TILT_DEGREE = 150
+    """The maximum value that the camera can tilt."""
+    MIN_PAN_TILT_DEGREE = 0
+    """The minimum value that the camera can pan and tilt."""
+    MAX_PAN_CALI_DEGREE = 180
+    """The maximum value to calibrate pan."""
+    MAX_TILT_CALI_DEGREE = 150
+    """The maximum value to calibrate tilt."""
+    MIN_PAN_CALI_DEGREE = -180
+    """The minimum value to calibrate pan."""
+    MIN_TILT_CALI_DEGREE = -150
+    """The minimum value to calibrate tilt."""
+
+
+    def __init__(self, bus_number=1, pan_channel=1, tilt_channel=2,
+                                                    camera_type=ABSOLUTE):
+        # The value to set Servo object offset to.
+        offset = 0
+
+        self._db = filedb.fileDB("config")
+        self._pan_offset = int(self._db.get('pan_offset', default_value=0))
+        self._tilt_offset = int(self._db.get('tilt_offset', default_value=0))
+
+        self._pan_servo = Servo.Servo(pan_channel, bus_number=bus_number,
+                                                   offset=offset)
+        self._tilt_servo = Servo.Servo(tilt_channel, bus_number=bus_number,
+                                                     offset=offset)
+
+        self._camera_type = RELATIVE if camera_type is RELATIVE else ABSOLUTE
+
+        self._current_pan_angle = 90
```

Comment:
Can you add a function that resets the values to this as well?

i.e. Camera.reset_camera()